Vagrantfile

```diff
@@ -3,4 +3,8 @@
 
 Vagrant.configure("2") do |config|
   config.vm.box = "multidev.box"
+
+  # Uncomment this line if you need to re-run the user provisioning script
+  # without re-building the box.
+  # config.vm.provision "shell", path: "user_provision.sh", privileged: false
```

Comment:
is this. equivalent to `vagrant up --provision`?

Reply:
Pretty much. You can also run `vagrant provision` to run it inside a running instance without restarting, etc. That's why I added all of the `rm` commands to `user_provision.sh`. That way I can re-run the script and it will run as close as possible to how it would run during the main build steps.